docs/resources/device_posture_rule.md

```diff
@@ -98,21 +98,22 @@ Optional:
 - `version` (String) The operating system semantic version.
 - `version_operator` (String) The version comparison operator for crowdstrike. Available values: `>`, `>=`, `<`, `<=`, `==`.
 
-
-<a id="nestedblock--match"></a>
-### Nested Schema for `match`
+<a id="nestedblock--input--locations"></a>
+### Nested Schema for `input.locations`
 
 Optional:
 
-- `platform` (String) The platform of the device. Available values: `windows`, `mac`, `linux`, `android`, `ios`, `chromeos`.
+- `paths` (Set of String) List of paths to check for client certificate rule.
+- `trust_stores` (Set of String) List of trust stores to check for client certificate rule. Available values: `system`, `user`.
```

Comment:
@aw-cf Can we be more specific about which `paths` and `trust_stores` options are available for each platform?

Reply:
Yes, I think we should be specific here. Please check the dashboard UI in regards to which combinations are valid.

Brief summary:
* Windows
  * `trust_stores`: `system`, `user`
* macOS
  * `trust_stores`: `system`
* Linux
  * `paths`
  * `trust_stores`: `system`

Reply:
Is it intentional that these are added in this file as well as `docs/resources/zero_trust_device_posture_rule.md`?
(Especially since other new attributes like `extended_key_usage` are not mentioned on this page)